src/apps/applib/lib/utility.c

```diff
@@ -68,12 +68,12 @@ int parseArgs(int argc, char* argv[], int numArgs, Arg* args[],
     const char* errorMessage = NULL;
     const char* errorDetails = NULL;
 
-    int failed = _parseArgsList(argc, argv, 4, args, helpArg, &errorMessage,
```

Comment:
This was a bug from the last diff?

Reply:
Yes, this was a copy paste error when refactoring the code. I found it because it from a segfault when testing another filter.